sql/core/src/test/scala/org/apache/spark/sql/DatasetSuite.scala

```diff
@@ -440,14 +439,16 @@ class DatasetSuite extends QueryTest with SharedSQLContext {
     implicit val kryoEncoder = Encoders.kryo[KryoData]
     val ds = Seq(KryoData(1), KryoData(2)).toDS()
 
-    val expectedAnswer = """+-----------+
```

Comment:
I'd like to remove this test entirely as we are not showing the real object content now.